pkg/controller/user/user_controller.go

```diff
@@ -483,6 +523,88 @@ func (c *Controller) ldapSync(user *iamv1alpha2.User) error {
 	}
 }
 
+func (c *Controller) deleteRoleBindings(user *iamv1alpha2.User) error {
+
```

Comment:
This doesn't improve the situation, please use `DeleteCollection` in clientset, it's more efficient

Reply:
fixed